src/shared/components/oncoprint/DeltaUtils.ts

```diff
@@ -662,6 +662,7 @@ function transitionClinicalTrack(
             },
             sort_direction_changeable: true,
             tooltipFn: makeClinicalTrackTooltip(nextSpec, true),
+            important_ids: nextSpec.altered_uids,
```

Comment:
maybe a comment why they are important?  Could it be for reason other than they are altered?

Reply:
in the oncoprint library I have a comment explaining this - important means they determine which rules show up in the legend. it could be any list of ids at all

Reply:
ill add a comment explaining that